pkg/cluster/hive.go

```diff
@@ -7,7 +7,7 @@ import (
 	"context"
 	"encoding/json"
 
-	"github.com/openshift/installer/pkg/asset/installconfig/azure"
+	icazure "github.com/openshift/installer/pkg/asset/installconfig/azure"
```

Comment:
It is an example of the rule which has not been enforced:

https://github.com/Azure/ARO-RP/blob/ed8ab351eaefb151c5f0f1e6589aab290c5fef68/hack/validate-imports/allowed-import-names.yaml#L49-L50

---

Similar thing can happen with regex rules. For example this one:

https://github.com/Azure/ARO-RP/blob/ed8ab351eaefb151c5f0f1e6589aab290c5fef68/hack/validate-imports/validate-imports.go#L144-L147

If you import like this:

```go
operatorfake1 "github.com/openshift/client-go/operator/clientset/versioned/fake"
```

It will be flagged by `validate-imports`. But if you import without an alias, it will not be flagged:

```go
"github.com/openshift/client-go/operator/clientset/versioned/fake"
```